rllib/offline/offline_prelearner.py

```diff
@@ -179,6 +188,9 @@ def __call__(self, batch: Dict[str, np.ndarray]) -> Dict[str, List[EpisodeType]]
                 )
                 for state in batch["item"]
             ]
+            # Ensure that all episodes are done and no duplicates are in the batch.
```

Comment:
dumb question: Why would we expect duplicates at all here? Any algorithm that would be in danger of running into something like this?

Reply:
Great question: This is tricky. I just ran into it because in my own projects I was using small datasets for testing. It could be that your data is so small that pulling a batch contains mjultiple duplicates. Let's say you have a dataset of size 10 and you pull a batch of size 100 (silly maybe, but if you have episodes one row could be multiple timesteps) now 90 rows would be duplicates in the batch.